components/VM_Arm/src/main.c

```diff
@@ -620,6 +620,29 @@ static void irq_handler(void *data, ps_irq_acknowledge_fn_t acknowledge_fn, void
     }
 }
 
+/* Default RAM initialization. Modules can overwrite this weak function with a
+ * custom initialization. e.g. to use memory shared with other components or
+ * mapped on demand.
+ */
+WEAK int init_ram_module(vm_t *vm)
+{
+    if (0 == ram_size) {
+        ZF_LOGW("VM%d: no RAM defined", vm->vm_id);
```

Comment:
Perhaps we should be consistent with existing code and not prefix the message here with `VM%d: `. I once took a look at ZF logging framework and it appears it could be possible to prefix all ZF log messages with some string. While I agree it would be nice to know from which VM the log message is, perhaps we should do it for all messages with that prefix thing?